Add getMediaStream function to attach MediaStream

marcusotterstad opened the pull request:
### Overview
I've been using OvenLiveKit-Web and noticed that there's no direct way to attach an external or pre-existing MediaStream. In the current setup, developers are limited to obtaining the media stream through getUserMedia or getDisplayMedia. To cater to diverse use-cases like mine, I've added a function called getMediaStream.

### Related Issue
This pull request is in response to the issue [#15](https://github.com/AirenSoft/OvenLiveKit-Web/issues/15) that I previously opened.

### How It Works
The getMediaStream function allows developers to pass an existing MediaStream. The internal mechanism then sets this MediaStream as the source for broadcasting, allowing for more flexible media input handling.

### Testing Done
Tested attaching an external MediaStream and broadcasting, which is working as expected.
Ensured existing getUserMedia and getDisplayMedia functionalities were unaffected.

SangwonOh commented:
@marcusotterstad Hi. Thanks for your contribution!  How about renaming the function name `getMediaStream` to `setMediaStream` or `attachMediaStream`? 👍

marcusotterstad commented:
@SangwonOh Thank you for feedback, I added a new commit where I changed it to setMediaStream. I thought setMediaStream was a better fit since attachMediaStream may be confused with the attachMedia function. If there are any other issues or improvements I can make, please let me know :)

SangwonOh reviewed (commented):
Thanks!

SangwonOh commented:
@marcusotterstad Thanks! 

> If there are any other issues or improvements I can make, please let me know :)

Ok! very appreciated
